src/hotspot/share/gc/g1/g1OopClosures.hpp

```diff
@@ -112,17 +112,17 @@ class G1ScanEvacuatedObjClosure : public G1ScanClosureBase {
 };
 
 // RAII object to properly set the _scanning_in_young field in G1ScanEvacuatedObjClosure.
-class G1ScanInYoungSetter : public StackObj {
+class G1SkipCardEnqueueSetter : public StackObj {
   G1ScanEvacuatedObjClosure* _closure;
 
 public:
-  G1ScanInYoungSetter(G1ScanEvacuatedObjClosure* closure, bool new_value) : _closure(closure) {
-    assert(_closure->_scanning_in_young == G1ScanEvacuatedObjClosure::Uninitialized, "Must not be set");
-    _closure->_scanning_in_young = new_value ? G1ScanEvacuatedObjClosure::True : G1ScanEvacuatedObjClosure::False;
+  G1SkipCardEnqueueSetter(G1ScanEvacuatedObjClosure* closure, bool skip_enqueue_cards) : _closure(closure) {
+    assert(_closure->_skip_card_enqueue == G1ScanEvacuatedObjClosure::Uninitialized, "Must not be set");
+    _closure->_skip_card_enqueue = skip_enqueue_cards ? G1ScanEvacuatedObjClosure::True : G1ScanEvacuatedObjClosure::False;
```

Comment:
There's no reason for `class G1SkipCardEnqueueSetter`, field `_skip_card_enqueue` and method arg `skip_enqueue_cards` to not follow the same pattern; both alternatives are fine, "skip enqueue cards" or "skip card enqueue".